app/views/apis/index.html.erb

```diff
@@ -1 +1,11 @@
 API Index Page
+
+<div class="apis">
+  <% @apis.each do |api| %>
+    <li>
```

Comment:
Lol github changed my markup to embed in the comment. Use an unordered or ordered list HTML element to enclose a list item.